src/wp-admin/js/inline-edit-post.js

```diff
@@ -100,27 +148,36 @@ inlineEditPost = {
 			}
 		});
 
+		// If no boxes where checked, just hide the quick/bulk edit rows.
```

Comment:
Please keep naming consistent. `boxes` should be `checkboxes`.

Also `where` should be `were`.